src/submission/forms.py

```diff
@@ -161,8 +163,10 @@ def __init__(self, *args, **kwargs):
                             widget=forms.TextInput(attrs={'div_class': element.width}),
                             required=element.required)
                     elif element.kind == 'textarea':
-                        self.fields[element.name] = forms.CharField(widget=forms.Textarea,
-                                                                    required=element.required)
+                        self.fields[element.name] = forms.CharField(
```

Comment:
Probably not. Making it a `BleachField` was the important bit here to avoid XSS but we can defer turning this into TinyMCE widget until we do the rest of the system.